### PR TITLE
Sep027 - type in Activity

### PR DESCRIPTION
**Pull request by eoberortner**

### CODE REVIEW

File: apdx-validation.tex [hunk 1]
```diff
@@ -811,6 +813,11 @@ \subsubsection*{Rules for the \class{Activity} class}
 \printModeling{An \sbol{Activity} with a child \sbol{Association} that has a \sbolmult{roles:A}{roles} property that contains the \sbol{URI}~\url{http://sbols.org/v2\#learn} SHOULD NOT have child \sbol{Usage} objects that have \sbolmult{roles:U}{roles} properties that contain the \sbol{URI}s~\url{http://sbols.org/v2\#design} or~\url{http://sbols.org/v2\#build}.\\ Reference: \sec{sec:Activity}}
 }
 
+\twothreezero{
+%% EO: using the \hyperref here due to the specification of the \sbol command and the "types" property that appears in ComponentDefinition and Activity.
+\printValid{The \texttt{\hyperref[sec:activity_types]{types}} property of an \sbol{Activity} object is OPTIONAL and MAY contain a set of \sbol{URIs}.  \\ Reference: \sec{sec:Activity}}
```
Comment (palchicz) [inline apdx-validation.tex]: The URIs link is broken here

File: apdx-validation.tex [hunk 2]
```diff
@@ -783,7 +783,9 @@ \subsubsection*{Rules for the \class{GenericTopLevel} class}
 \subsubsection*{Rules for the \class{Activity} class} 
 \setcounter{sbolCtr}{12401}
 
-\printValid{An \sbol{Activity} MUST NOT have properties other than the following: \sbol{identity},\\ \sbol{persistentIdentity}, \sbol{displayId}, \sbol{version}, \sbol{wasDerivedFroms}, \sbol{wasGeneratedBys}, \sbol{name},\\ \sbol{description}, \sbol{annotations}, \sbol{startedAtTime}, \sbol{endedAtTime}, \sbol{wasInformedBys},\\ \sbol{usages}, and \sbol{associations}.\\ Reference: \sec{sec:Activity}}
+\twothreezero{
+\printValid{An \sbol{Activity} MUST NOT have properties other than the following: \sbol{identity},\\ \sbol{persistentIdentity}, \sbol{displayId}, \sbol{version}, \sbol{wasDerivedFroms}, \sbol{wasGeneratedBys}, \sbol{name},\\ \sbol{description}, \sbol{annotations}, \sbol{types}, \sbol{startedAtTime}, \sbol{endedAtTime}, \sbol{wasInformedBys},\\ \sbol{usages}, and \sbol{associations}.\\ Reference: \sec{sec:Activity}}
```
Comment (palchicz) [inline apdx-validation.tex]: The `types` link is broken here

File: practices.tex [hunk 3]
```diff
@@ -179,17 +179,23 @@ \subsubsection{Activity}
...
 
-\paragraph{The \sbolheading{associations} property}\label{sec:associations}
-The \sbol{associations} property is OPTIONAL and MAY contain a set of \sbol{URI}s that refers to \sbol{Association} objects.
+\twothreezero{
+\paragraph{The \sbolheading{types} property}\label{sec:activity_types}
+The \sbol{types} property is an OPTIONAL set of \sbol{URI}s that explicitly specify the type of the provenance \sbol{Activity} in more detail.
```
Comment (palchicz) [inline practices.tex]: types link is broken here

**Comment by eoberortner** [comment]
please review the last two submits (c76567c and 1892577) addressing the requested changes

**Pull request merged**